lib/sht4x/measurement.ex

```diff
@@ -23,6 +24,27 @@ defmodule SHT4X.Measurement do
   this is real temperature and humidity detected.
   """
   @spec from_raw(<<_::48>>) :: t()
+  def from_raw(<<raw_t::16, _crc1, raw_rh::16, _crc2>>)
+      when raw_t == 0x8000 and raw_rh == 0x8000 do
+    timestamp_ms = System.monotonic_time(:millisecond)
+    temperature_c = temperature_c_from_raw(raw_t)
+    humidity_rh = humidity_rh_from_raw(raw_rh)
+
+    Logger.warning(
+      "Your SHT4X sensor is returning values that could indicate the sensor has been damaged by ESD!"
```

Comment:
```suggestion
      "Your SHT4X sensor is returning values that could indicate the sensor has been damaged!"
```

I'm slightly skeptical on the ESD part, but the sensor definitely has some damage so that part seems good to report.